src/interface/report/TimeEventFilter.ts

```diff
@@ -1,9 +1,23 @@
-import React from 'react';
+import React, { ReactNode } from 'react';
 
 import { captureException } from 'common/errorLogger';
 import { SECOND_POTIONS } from 'parser/shared/modules/items/PrePotion';
 import Fight from 'parser/core/Fight';
-import { EventType, Event, PhaseEvent, CastEvent, ApplyBuffStackEvent, ApplyDebuffStackEvent, RemoveBuffStackEvent, RemoveDebuffStackEvent, ApplyBuffEvent, RemoveBuffEvent, ApplyDebuffEvent, RemoveDebuffEvent, FilterCooldownInfoEvent } from 'parser/core/Events';
+import {
+  EventType,
+  PhaseEvent,
+  CastEvent,
+  ApplyBuffStackEvent,
+  ApplyDebuffStackEvent,
+  RemoveBuffStackEvent,
+  RemoveDebuffStackEvent,
+  ApplyBuffEvent,
+  RemoveBuffEvent,
+  ApplyDebuffEvent,
+  RemoveDebuffEvent,
+  FilterCooldownInfoEvent,
+  AnyEvent,
+} from 'parser/core/Events';
```

Comment:
I hate this 😭 is this going to be required for lint?

Reply:
No